src/components/todo-item.vue

```diff
@@ -1,7 +1,20 @@
 <template>
   <li>
-    <label>{{todo}}</label>
-    <button @click="$emit('removeTodoItem')">Remove</button>
+    <div v-if="!todo.editing" @click="editMode(todo)">
+      {{todo.title}} 
+      <button @click="$emit('editTodo',editTodoCache)">Edit</button>
```

Comment:
editTodoCache <- please change naming to correct one

Reply:
editTodoCache -> beforeEditingValue